stable/database/files/nuobackup

```diff
@@ -264,8 +271,15 @@ do
 done
 
 backup_type=$(echo $backup_type | tr '[:upper:]' '[:lower:]')
-# include all HCSMs if no labels are provided
-[ -z "$labels" ] && labels="role hotcopy"
+# determine the HCSMs selector based on the input
+if [ -n "$labels" ]; then
+   selector+=("--labels" "$labels")
+elif [ -n "$process_filter" ]; then
```

Comment:
Do you want to throw an error or print a warning if the user provides both a label and a filter?

Reply:
`nuodocker backup database` enforces mutual exclusion, but this does suppress that check and only supply one of the two supplied arguments, which we don't want to do.

I would change this so that we plumb everything through to `nuodocker backup database` and let it decide what to do with the arguments. That could be done by replacing `elif` with `if` and explicitly checking the `selectors` array when deciding whether to inject the default.